<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BillItemGUI</class>
 <widget class="QWidget" name="BillItemGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dati voce di computo con misure</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="dataTab">
      <attribute name="title">
       <string>Dati</string>
      </attribute>
      <layout class="QGridLayout" name="dataTabLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="priceGroupBox">
         <property name="title">
          <string>Prezzo</string>
         </property>
         <layout class="QGridLayout" name="priceDataLayout">
          <item row="0" column="1">
           <widget class="QLineEdit" name="priceCodeLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="priceUnitMeasureLabel">
            <property name="text">
             <string>Unità di misura</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="priceShortDescLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="priceCodeLabel">
            <property name="text">
             <string>Codice</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="priceShortDescLabel">
            <property name="text">
             <string>Descrizione</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="priceUnitMeasureLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="amountsGroupBox">
         <property name="title">
          <string>Importi</string>
         </property>
         <layout class="QGridLayout" name="amountsDataLayout"/>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="valueDataGroupBox">
         <property name="title">
          <string>Quantità</string>
         </property>
         <layout class="QGridLayout" name="valueDataLayout">
          <item row="0" column="1">
           <widget class="QLabel" name="quantityLabel">
            <property name="text">
             <string>Quantità</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="quantityLineEdit">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QCheckBox" name="associateLinesCheckBox">
            <property name="text">
             <string>Associa misure</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="3">
           <widget class="QGroupBox" name="billItemLinesGroupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Misure</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <widget class="QPushButton" name="addBillItemLinePushButton">
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="delBillItemLinePushButton">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="importBillItemMeasuresTXTPushButton">
               <property name="text">
                <string>TXT</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="3">
              <widget class="QTableView" name="billItemLinesTableView"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="priceTab">
      <attribute name="title">
       <string>Prezzo</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5"/>
     </widget>
     <widget class="QWidget" name="attributeTab">
      <attribute name="title">
       <string>Etichette</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0">
        <widget class="QPushButton" name="addAttributePushButton">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="removeAttributePushButton">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QTableView" name="attributeTableView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
